test/serialize.test.ts

```diff
@@ -0,0 +1,236 @@
+// 500k arbitrary max bundle size. Some functions may need more.
+
+import { Stack } from "aws-cdk-lib";
+import { AttributeType } from "aws-cdk-lib/aws-dynamodb";
+import axios from "axios";
+import { v4 } from "uuid";
+import { serialize, bundle, $AWS, Table } from "@fnls";
+
+// In that case increase explicitly.
+const BUNDLED_MAX_SIZE = 500 * 1024;
```

Comment:
Is this excessive? I know we will slowly improve this over time, but 500KB seems excessive for the tests I see below.

Reply:
Can lower it, was arbitrary

Reply:
lowered to 50k with the `Aaios` test using 500k as it is 300k bundled.

Reply:
Axios is 300k bundled? gah-dam